code/modules/mob/living/simple_animal/friendly/cat.dm

```diff
@@ -331,3 +331,20 @@
 	icon_dead = "penny_dead"
 	icon_rest = "penny_rest"
 	holder_type = /obj/item/holder/cat/penny
+/mob/living/simple_animal/cat/crusher // Goodbye Runtime, make way for the new girl in town.
```

Comment:
```suggestion
/mob/living/simple_animal/cat/crusher
```

Reply:
no fun allowed when sleepygemmy is reviewing!

Reply:
clean code my beloved...

Reply:
No